frontend/src/map/lulcColors.js

```diff
@@ -0,0 +1,258 @@
+export default [
+  "#000000",
+  "#000000",
```

Comment:
Oh good catch. Turns out there are 4 Alaska-only types in NLCD and I guess the color palette already excluded them. Also I forgot to add the `src/lulcCodes.js` file.

Reply:
But yes, the pixel values are used to index into this array of colors, so that's why we need such a long array even when there are only 16 unique landuse codes. The alternative to this would be create a multiband raster with the rgb values we want and serve & render that directly, instead of doing this client-side styling. But we also might want/need the integer landuse codes available to the client for other reasons.

Reply:
I agree it makes sense to preserve the LULC codes and style on the client side

Reply:
Could also consider storing the each LULC color, name, and code together -the values in lulcCodes could look like
```
  11: {name: 'Open Water', color: '#466b9f'}
```
then iterate through that to make the colors array

Reply:
Ahh yes, much better! Please see `lulcCodes.js` renamed to `landuseCodes.js` (just easier to say/read). And then I build the colors array in `lulcLayer.js` so I removed `lulcColors.js`.